### ISSUE TITLE
Make model typed (maybe use ark-ts?)

### ISSUE THREAD

**Issue by Nasicus** [ArkEcosystem/ark-explorer#43]
While doing my PR's I noticed that even though we use typescript a lot of stuff is not yet typed.
Especially all Ark model classes (blocks, users, transactions, addresses, etc.).

I'd like to make them typed.

This would be no problem, however I saw that there is already an ark-ts project:
https://github.com/ArkEcosystem/ark-ts

This project does not only define types but also offers methods to access the API.

Why is this project not used in ark-explorer?

I also noticed that the "model" and the api is not identical to the ark-explorer.

For example:

To get transactions:
Ark-Explorer: `/getLastTransactions`
Ark-ts: `/transaction`

Block model - number of transactions:
Ark-Explorer: `block.transactionsCount`
Ark-ts: `block.numberOfTransactions`

So in summary I have two questions:
- Should I make the model typed in ark-explorer?
- What is the connection / difference to `ark-ts` (and the used API)?

Sorry for the long post :)
